docs/glossary/glossary.md

```diff
@@ -1,5 +1,12 @@
 # Splinter Glossary
 
+## Canopy
+
+Application interface for Splinter that can distribute apps dynamically.
```

Comment:
"distribute" doesn't seem like the right word here. "present" maybe?

Reply:
@jsmitchell - Initial info from @vaporos used the phrases "distributed apps" and "distributes apps".  Can you confer to pick a word or phrase you're both okay with?

P.S.  If it's "present" - I'd change that to "provide" (because "present" is also a noun).

Reply:
I think provide is appropriate. The UI plugins are not "distributed apps" in the distributed ledger sense of the word.

Reply:
Changed to "provide".